### PR TITLE
prep-html-script-tag - Blythe Hill

### PR DESCRIPTION
**Pull request by blythehill**
Screenshot 2024-03-02 at 12 58 26 PM

### CODE REVIEW

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Hey Blythe, your files were not added to the branch. make sure to add and commit them before pushing again.

its likely because they weren't saved

File: prep-html-script-tag/index.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    <title>HTML Script Tag</title>
+  </head>
+  <body>
+    <script>
+      console.log('I have added JS to a script tag in the HTML!');
+    </script>
+    <script src="main.js"></script>
+    <script>
+      console.log('I have added a script file via the src attribute!');
+    </script>
```
Comment (BrettAlbright) [inline prep-html-script-tag/index.html]: this code, minus the script tag, should be in the main.js file

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
Make sure the second console log statement is in the main.js file

the script tag with the src attribute will load the code from main.js into the index.html file

File: prep-html-script-tag/index.html (hunk 1 above)
Comment (blythehill) [reply prep-html-script-tag/index.html]: <script src="main.js" console.log('I have added a script file via the src attribute!');></script>
 
Is this correct? I'm not sure how to place the console.log code "in" the main.js file...

**Pull request closed without merging**